server/interceptor/interceptor_test.go

```diff
@@ -68,6 +68,7 @@ func TestTrillianInterceptor_TreeInterception(t *testing.T) {
 		handlerErr error
 		wantErr    bool
 		wantTree   *trillian.Tree
+		cancelled  bool
```

Comment:
nit: I think a `ctx` parameter would be better/more flexible than `cancelled`. We can have a pre-created cancelled ctx for this test.

Reply:
Not sure that's correct. We'd then have to manage not creating / cancelling the other test contexts. If you want to look at refactoring the interceptor code to make it easier to test cancellation then that's another option.